### PR TITLE
Gus/dashboard over update

### PR DESCRIPTION
**Pull request by l0k0ms**
Updating tagging guide for dashboard to answer this issue:
https://github.com/DataDog/documentation/issues/1381

### CODE REVIEW

File: content/guides/tagging.md [hunk 1]
```diff
@@ -117,16 +117,26 @@ After you have assigned tags at the host and integration level, you can start us
...
 
 {{< img src="eventtags.png" alt="Events List and Tags" >}}
 
 ### Using tags in Dashboards
-You can use tags to narrow down the metrics to display on a dashboard grapm, or to create groups of metrics to display. To narrow down the metrics to display, enter the tag in the over: textbox. You will now be looking at a chosen metric over all the hosts that have that particular tag assigned. To group using tags, enter the key part of the tag in the group: textbox. For instance, if you have a timeseries graph and you have assigned the tags ```role:database```, ```role:frontend```, and ```role:loadbalancer```, you will get one line in your timeseries graph representing all the machines with the database, another of machines wth the frontend, and third of machines with the loadbalancer.
 
-{{< img src="dashboardtags.png" alt="Tags in Dashboards" >}}
+You can use tags to narrow down the metrics to display on a dashboard grapm, or to create groups of metrics to display.
```
Comment (kmshultz) [inline content/guides/tagging.md]: s/grapm/graph

File: content/guides/tagging.md [hunk 2]
```diff
@@ -117,16 +117,26 @@ After you have assigned tags at the host and integration level, you can start us
...
 
 {{< img src="eventtags.png" alt="Events List and Tags" >}}
 
 ### Using tags in Dashboards
-You can use tags to narrow down the metrics to display on a dashboard grapm, or to create groups of metrics to display. To narrow down the metrics to display, enter the tag in the over: textbox. You will now be looking at a chosen metric over all the hosts that have that particular tag assigned. To group using tags, enter the key part of the tag in the group: textbox. For instance, if you have a timeseries graph and you have assigned the tags ```role:database```, ```role:frontend```, and ```role:loadbalancer```, you will get one line in your timeseries graph representing all the machines with the database, another of machines wth the frontend, and third of machines with the loadbalancer.
 
-{{< img src="dashboardtags.png" alt="Tags in Dashboards" >}}
+You can use tags to narrow down the metrics to display on a dashboard grapm, or to create groups of metrics to display. 
+To narrow down the metrics to display, enter the tag in the ```from:```textbox.
```
Comment (kmshultz) [inline content/guides/tagging.md]: Missing a space between `from` and "textbox"

File: content/guides/tagging.md [hunk 3]
```diff
@@ -117,16 +117,26 @@ After you have assigned tags at the host and integration level, you can start us
...
 
-{{< img src="dashboardtags.png" alt="Tags in Dashboards" >}}
+You can use tags to narrow down the metrics to display on a dashboard grapm, or to create groups of metrics to display. 
+To narrow down the metrics to display, enter the tag in the ```from:```textbox. 
+You will now be looking at a chosen metric over all the hosts that have that particular tag assigned.
+
+{{< img src="dashboardtags_1.png" alt="Tags in Dashboards from textbox" >}}
+
+To group using tags, enter the key part of the tag in the ```avg by:``` textbox. 
+For instance, if you have a timeseries graph and you have assigned the tags `role:database`, `role:frontend`, and `role:loadbalancer`, you will get one line in your timeseries graph representing all the machines with the database, another of machines wth the frontend, and third of machines with the loadbalancer.
```
Comment (kmshultz) [inline content/guides/tagging.md]: s/wth/with

Comment (kmshultz) [inline content/guides/tagging.md]: > and third of machines

and a third of machines.

Also, in this passage let's replace 'machines' with 'hosts'

Comment (kmshultz) [inline content/guides/tagging.md]: > +For instance, if you have a timeseries graph and you have assigned the tags `role:database`, `role:frontend`, and `role:loadbalancer`, you will get one line in your timeseries graph representing all the machines with the database, another of machines wth the frontend, and third of machines with the loadbalancer.

Let's make the example crystal clear:

"For instance if you have a time series graph showing a metric tagged variously with `role:database`, `role:frontend`, or `role:loadbalancer`, enter `role` in the **avg_by** textbox. This causes the graph to show just one line for each tag value—`database`, `frontend`, and `loadbalancer`. Each line represents the average metric value across all hosts sharing that tag value."

File: content/guides/tagging.md [hunk 4]
```diff
@@ -117,16 +117,26 @@ After you have assigned tags at the host and integration level, you can start us
...
+
+{{< img src="dashboardtags.png" alt="Tags in Dashboards avgby textbox" >}}
 
-You can also use tags to overlay events on the dashboard. This works in exactly the same way as in the Events List. Simply enter ```tags:``` followed by the tag and you will see the corresponding events overlaid as vertical bars on each graph.
+You can also use tags to overlay events on the dashboard. This works in exactly the same way as in the Events List. 
+Simply enter ```tags:``` followed by the tag and you will see the corresponding events overlaid as vertical bars on each graph.
```
Comment (kmshultz) [inline content/guides/tagging.md]: Let's move this up to the previous paragraph.

File: content/guides/tagging.md [hunk 5]
```diff
@@ -136,7 +146,7 @@ To filter the list of hosts in the Infrastructure list, enter a tag in the filte
 
 ### Using tags in Monitors
 
-When defining a monitor, you can use tags to allow the monitor to apply to any subset of hosts across your environment.
+When defining a monitor, you can use tags in the ```from:``` textbox.  to allow the monitor to apply to any subset of hosts across your environment.
```
Comment (kmshultz) [inline content/guides/tagging.md]: > When defining a monitor

When creating (or editing) a monitor

Comment (kmshultz) [inline content/guides/tagging.md]: > to allow the monitor to apply to any subset of hosts across your environment.

"to limit the monitor's scope."

(it's not just for limiting hosts; not all metrics are per-host)

File: content/guides/tagging.md (hunk 4 above)
Comment (l0k0ms) [inline content/guides/tagging.md]: I would let it under the screenshot since the screenshot is directly related to the previous paragraph

**Pull request merged**